src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/iOS/PopupRenderer.ios.cs

```diff
@@ -148,25 +148,38 @@ void SetLayout()
 			{
 				var originY = Element.VerticalOptions.Alignment switch
 				{
-					LayoutAlignment.End => UIScreen.MainScreen.Bounds.Height - PreferredContentSize.Height,
-					LayoutAlignment.Center => (UIScreen.MainScreen.Bounds.Height / 2) - (PreferredContentSize.Height / 2),
+					LayoutAlignment.End => UIScreen.MainScreen.Bounds.Height,
+					LayoutAlignment.Center => UIScreen.MainScreen.Bounds.Height * 0.5,
```

Comment:
can you move this `0.5` to a constant? And replace all 0.5 for this constant?

Reply:
We can also use `xxx / 2`. This is mostly used here.

Reply:
my comment is to have those values in one place, that way is easier to change them if needed.

Reply:
You mean all 36 references in 16 files? Sure? 😅
![image](https://user-images.githubusercontent.com/6826752/107861770-3886ef80-6e48-11eb-8cfa-83e8c1e0c557.png)

Reply:
lol Didn't realize that is that much... Yeah, nevermind. This refactoring shouldn't be on this PR.